spec/abstractops.html

```diff
@@ -648,6 +638,24 @@ <h1>LargerOfTwoTemporalDurationUnits ( _u1_, _u2_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-mergelargestunitoption" aoid="MergeLargestUnitOption">
+    <h1>MergeLargestUnitOption ( _options_, _largestUnit_ )</h1>
+    <p>
+      The abstract operation MergeLargestUnitOption returns a new plain Object with the properties copied from the _options_ Object, and the `largestUnit` property set to _largestUnit_.
+    </p>
+    <emu-alg>
+      1. Let _merged_ be ! OrdinaryObjectCreate(%Object.prototype%).
+      1. Let _keys_ be ? _options_.[[OwnPropertyKeys]]().
+      1. For each element _nextKey_ of _keys_, do
+        1. Let _desc_ be ? _options_.[[GetOwnProperty]](_nextKey_).
+        1. If _desc_ is not *undefined* and _desc_.[[Enumerable]] is *true*, then
```

Comment:
if you’re already filtering to own enumerable properties, and if symbol keys aren’t needed, could this whole algorithm use https://tc39.es/ecma262/#sec-enumerableownpropertynames ?

another option is to call `%Object.assign%` directly, since that’s basically what this is doing?